witness/src/lib.rs

```diff
@@ -63,18 +66,25 @@ impl<T: Sized + Sync + Clone + Send + Copy + Default + PrimeCharacteristicRing>
             padding_strategy: InstancePaddingStrategy::Default,
         }
     }
+
     /// convert into the p3 RowMajorMatrix, with padded to next power of 2 height filling with T::default value
-    pub fn into_default_padded_p3_rmm(self) -> p3::matrix::dense::RowMajorMatrix<T> {
+    /// padding its height to the next power of two (optionally multiplied by a `blow_factor`)
+    /// padding is filled with `T::default()`, and the transformation consumes `self`
+    pub fn into_default_padded_p3_rmm(
+        mut self,
+        blow_factor: Option<usize>,
```

Comment:
nitpick: shall we call it `blowup_factor`?

Reply:
👍 , I made few renaming in this commit and update https://github.com/scroll-tech/ceno/pull/933/commits/44665dd01729ee7b1c2280bae2a65ee36b329fdf